app/serializers/web_hook_post_serializer.rb

```diff
@@ -14,4 +17,9 @@ def include_topic_title?
       false
     end
   end
+
+  def topic_posts_count
+    object.topic && object.topic.posts_count
```

Comment:
I'm thinking that it might be better to return `0` here when `object.topic` is `nil`.

Reply:
Sure, that makes sense.